crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -268,10 +268,18 @@ def run_installer
       def restore_database
         Rails.logger.debug "Restoring Crowbar database"
         begin
+          Crowbar::Migrate.migrate!
           SerializationHelper::Base.new(YamlDb::Helper).load(
             @data.join("crowbar", "database.yml")
           )
         rescue SQLite3::SQLException => e
+          Rails.logger.debug "Failed to migrate database: #{e}"
+          set_failed
+          @status[:restore_database] = {
+            status: :not_acceptable,
+            msg: I18n.t("backups.index.migrate_database_failed")
+          }
+        rescue StandardError => e
```

Comment:
maybe I have to be more specific here now after putting everything in one begin block... If Migrate throws another error than SQLite, we will have a misleading log entry